neuro_extras/main.py

```diff
@@ -93,7 +93,9 @@ def _create_builder_container(
         image_ref: str,
         build_args: Sequence[str] = (),
     ) -> neuro_api.Container:
-        command = f"--dockerfile={dockerfile_path} --destination={image_ref}"
+        command = (
+            f"--dockerfile={dockerfile_path} --destination={image_ref} --cache=true"
```

Comment:
I am not sure this works.
we do not specify `--cache-dir` anywhere neither a storage volume.
as far as I get it, if we rely on `--cache-repo` which is inferred from `--destination`, then we cannot use cache of base-image layers for repositories named differently.

Reply:
`--cache-dir` is only used for base image layers. `--cache=true` enables intermediate layer caching in the destination repository.
`--cache-dir` for base images requires cache pre-population as we discussed yesterday.

Reply:
@dalazx the code has been updated to use per-user cache